app/celer-sim/Runner.cc

```diff
@@ -159,22 +159,24 @@ size_type Runner::num_events() const
 /*!
  * Get the accumulated action times.
  *
- * Action times are only collected by the transporter when running with a
- * single stream.
+ * This is a *mean* value over all streams.
  */
-auto Runner::get_action_times() -> MapStrReal
+auto Runner::get_action_times() -> MapStrDouble
```

Comment:
```suggestion
auto Runner::get_action_times()  const -> MapStrDouble
```

Reply:
Ah actually because this access the transporters, which were non-const because they did lazy construction. I've amended with a const-correct accessor.